crm_stage_probability/views/crm_lead.xml

```diff
@@ -7,14 +7,14 @@
         <field name="arch" type="xml">
             <xpath
                 expr="//button[@name='action_set_automated_probability']"
-                position="before"
+                position="after"
```

Comment:
I think this should stay before and should look like this
![image](https://github.com/OCA/crm/assets/1799080/6db66530-167c-4e91-9a86-88dbb05c9a46)